packages/wonder-blocks-data/src/util/request-fulfillment.js

```diff
@@ -76,13 +76,17 @@ export class RequestFulfillment {
                     delete handlerRequests[key];
                     /**
                      * Let's cache the data!
+                     *
+                     * NOTE: This only caches when we're server side.
                      */
                     return cacheData<TOptions, TData>(handler, options, data);
```

Comment:
This is because we only track the request fulfillments on the server, right?

Reply:
It's because we only cache for the server to make it so that we can render things on initial render.